rfcs/0175-appeals-council.md

```diff
@@ -0,0 +1,165 @@
+---
+feature: Memorandum of Understanding on Equitable Moderation
+start-date: 2024-04-24
+author: nrdxp
+co-authors: apcodes
+shepherd-team:
+shepherd-leader:
+related-issues:
+---
+
+# Summary
+[summary]: #summary
+
+- The primary objective of this RFC is to fundamentally enhance the existing moderation practices to ensure fairness and transparency within our community.
+
+- We propose the adoption of a *Memorandum of Understanding on Equitable Moderation* (hereinafter referred to as the *Memorandum*), establishing a foundation for reasonable expectations within a community of equal and reasonable contributors.
+
+- To prevent the abuse of exclusionary power inherent in community moderation, we aim to establish an *Appeals Council* with a diverse composition.
```

Comment:
Would you be satisfied if the moderation team couldn’t ban you from NixOS/nix{,pkgs}?

The more people you put into the bureaucracy, the more bureaucratic it will get.